spec/api-power-monitor-spec.js

```diff
@@ -41,7 +43,7 @@ const skip = process.platform !== 'linux' || !process.env.DBUS_SYSTEM_BUS_ADDRES
     before((done) => {
       logindMock.on('MethodCalled', onceMethodCalled(done))
       // lazy load powerMonitor after we listen to MethodCalled mock signal
-      powerMonitor = require('electron').remote.powerMonitor
+      dbusMockPowerMonitor = require('electron').remote.powerMonitor
```

Comment:
@tarruda Is it required to lazy initialize the module ? I was hoping we can just initialize the module only once.

Reply:
It has to be lazy loaded if we want to verify that electron inhibits the system suspend (it does this by calling a method via dbus).